main/webapp/modules/core/scripts/facets/list-facet.js

```diff
@@ -82,6 +84,7 @@ ListFacet.prototype.getJSON = function() {
   var o = {
       type: "list",
       name: this._config.name,
+      minimize: this._config.minimize,
```

Comment:
I don't think `minimize` should be part of the JSON serialization of the facet, since the backend does not care about whether a facet is minimized or not. I would keep this a local attribute of the facet, outside its `_config`.

Reply:
@wetneb: funny as it seams so obvious AFTER you write it… ;-)

I've fixed it as required.

Regards, A.